test/e2e/e2e.test.js

```diff
@@ -116,16 +121,39 @@ describe('the RTC Twilio-CLI Plugin', () => {
         stdout.start();
         await ViewCommand.run([]);
         stdout.stop();
-        expect(stdout.output).toMatch(/Web App URL: .+\nPasscode: \d{3} \d{3} \d{4} \d{4}\nExpires: .+/);
+        expect(stdout.output).toMatch(
+          /Web App URL: .+\nPasscode: \d{3} \d{3} \d{4} \d{4}\nExpires: .+\nRoom Type: group/
+        );
       });
     });
 
-    describe('the deploy command', () => {
-      it('should return a video token when the correct passcode is provided', async () => {
+    describe('the serverless deployment', () => {
```

Comment:
Can we add a test that validates the token is properly returned when the REST create fails?